src/test/java/in/zapr/druid/druidry/aggregator/DoubleMaxAggregatorTest.java

```diff
@@ -40,10 +40,13 @@ public void init() {
     }
 
     @Test
-    public void testAllFields() throws JsonProcessingException, JSONException {
```

Comment:
I think as we have deprecated and not removed implementation of old constructor, we should not remove their old test cases, could be marked as deprecated. Rather than replacing we could add new test cases.

Reply:
Reverted changes for this method.